Woof/WoofTests/Model/LocalStorage/LocalStorageDeleteTests.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+
+final class LocalStorageDeleteTests: XCTestCase {
+    private var localStorage: LocalStorage!
+    private let defaultValueToSave = "Hello, World!".data(using: .utf8)!
+
+    override func setUp() {
+        let storage = LocalStorage()
+        localStorage = storage
+
+        if let bundleIdentifier = Bundle.main.bundleIdentifier {
+            storage.localStorage?.removePersistentDomain(forName: bundleIdentifier)
+        }
+    }
+
+    override func tearDown() {
+        localStorage = nil
+        super.tearDown()
+    }
+
+    func testDeleteMethodExistsInAPI() {
+        localStorage.deleteValue(forKey: "key")
+    }
+
+    func testDeleteValueWithExistingKey() {
```

Comment:
nit:

Do we have an agreement to use the Present Simple tense in the names of tests?

Reply:
I will try to avoid these mistakes in the future